core/src/main/resources/hudson/widgets/HistoryWidget/index_pl.properties

```diff
@@ -20,8 +20,8 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-for\ all=Dla wszystkich
+for\ all=dla wszystkich
 for\ failures=dla nieudanych
-Clear=wyszyść
+Clear=wyczyść
```

Comment:
should this be lowercase?

Reply:
Honestly I have no idea as I haven't seen this action. I decided to fix typo which I found in repository :)
So maybe not perfect but still better than it was!

Reply:
Sure :)